.storybook/mocks/index.ts

```diff
@@ -0,0 +1,2 @@
+export { product } from './product'
```

Comment:
Nice! Could you please update it on Nextjs starter as well if it's not too much work? 
I noticed there we have mocks and mock folders.

Reply:
Yeah! I'm opening the PR!